Gitter/Gitter/Gitter.WindowsPhone/Views/AboutPage.xaml

```diff
@@ -0,0 +1,81 @@
+﻿<Page
+    x:Class="Gitter.Views.AboutPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    NavigationCacheMode="Required"
+    RequestedTheme="Light"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+    DataContext="{Binding About, Source={StaticResource Locator}}">
+
+    <!--LayoutRoot is the root grid where all page content is placed-->
+    <Grid x:Name="LayoutRoot" Background="Transparent">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="*"/>
+        </Grid.RowDefinitions>
+
+        <!--ContentPanel - place additional content here-->
+        <Grid x:Name="ContentPanel" Grid.Row="1" Margin="12,0,12,0">
+            <Hub Header="About">
+                <HubSection Header="Information">
+                    <DataTemplate>
+                        <ScrollViewer>
+                            <Grid>
+                                <Grid.RowDefinitions>
+                                    <RowDefinition Height="Auto"/>
+                                    <RowDefinition Height="Auto"/>
+                                    <RowDefinition Height="Auto"/>
+                                    <RowDefinition Height="*"/>
+                                    <RowDefinition Height="Auto"/>
+                                </Grid.RowDefinitions>
+
+                                <StackPanel Margin="0 0 0 24">
+                                    <TextBlock Text="{Binding ApplicationTitle}"
+                                           Style="{ThemeResource SubheaderTextBlockStyle}"/>
+                                    <TextBlock Text="{Binding ApplicationVersion}"
+                                           Style="{ThemeResource TitleTextBlockStyle}"/>
+                                </StackPanel>
+
+                                <TextBlock Grid.Row="1"
+                                       Text="{Binding ApplicationDescription}"
+                                       Style="{ThemeResource TitleTextBlockStyle}"
+                                       TextWrapping="Wrap" />
+                                <HyperlinkButton Grid.Row="2"
+                                             Content="Having Problems? Click to raise an issue"
+                                             Margin="0,12,0,0"
+                                             HorizontalAlignment="Left"
+                                             FontSize="17.333"
+                                             FontWeight="Bold"
+                                             FontFamily="Segoe WP"
+                                             NavigateUri="https://github.com/Odonno/Modern-Gitter/issues/new"/>
```

Comment:
https://github.com/Odonno/Modern-Gitter/issues/ will be better, some people are not logged in by default...